Table docs (#59395)
grafana/grafana: docs/sources/panels-visualizations/visualizations/table/index.md

You can use the table footer to show [calculations]({{< relref "../../calculation-types/" >}}) on fields.

After you enable the table footer:

1. Select the **Calculation**
2. Select the **Fields** that you want to calculate

The system applies the calculation to all numeric fields if you do not select a field.

### Count rows

> Select the **Count** calculation with **Count rows** enabled when you want to show the number of rows in the dataset instead of the number of values in the selected fields.

Reviewer: ```suggestion
If you want to show the number of rows in the dataset instead of the number of values in the selected fields, select the **Count** calculation and enable **Count rows**.
```
